src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.lib.Dao;
+import mate.academy.lib.DataProcessingException;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String URL = "jdbc:mysql://localhost:3306/your_schema?serverTimezone=UTC";
+    private static final String USER = "your_user";
+    private static final String PASSWORD = "your_password";
+
+    @Override
+    public Book create(Book book) {
+        String insertQuery = "INSERT INTO books (title, price) VALUES (?, ?)";
+        try (Connection connection = DriverManager.getConnection(URL, USER, PASSWORD);
+                PreparedStatement statement = connection.prepareStatement(insertQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {

```

Comment:
Use `PreparedStatement` over `Statement`, even for a static query with no parameters in `findAll()` method. It's the best practice, and it's slightly faster. Also, let's save each query in a separate variable prior to the try block for better readability and maintainability.